asm/macros/event.inc

```diff
@@ -2471,6 +2471,11 @@
 	callnative Script_SetDifficulty, requests_effects=1
 	.byte \difficulty
 	.endm
+    
+    .macro forcesave
+    callnative Script_ForceSaveGame
+    waitstate
+    .endm
```

Comment:
Indentation (needs tabs instead of spaces)

Reply:
done